### ISSUE TITLE
Edits to reports for after STAR review

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [iantaylor-NOAA/Lingcod_2021#129]
Place to document any needed/wanted changes to the reports that we find during the STAR review. This is first attempt so happy to  move this, ignore this, or use a different approach.

North
- [x] widen Table 14 and 15 so each row is on one line (@brianlangseth-NOAA )
- [x] Revise text on page 18 of north doc "Information from dockside interviews are thought ot be more reliable than information contained in the MRFSS database...." based on Theresa's comment during STAR that although this is true it would be OK to use MRFSS abundance index for future assessments (@kellijohnson-NOAA )
  - ~~Applying index for this may be best as future research recommendation~~

South

Both

- [ ] Include figure showing comparison of weighting by DM and Francis methods that was shown during STAR panel into the report (@iantaylor-NOAA )
- [x] Update text in sections marked with "manual updates": https://github.com/iantaylor-NOAA/Lingcod_2021/search?q=%22manual+updates%22 (@iantaylor-NOAA )

**Comment by kellijohnson-NOAA** [comment]
@brianlangseth-NOAA are you going to have time to fix the tables by 2021-07-30?

**Comment by brianlangseth-NOAA** [comment]
@kellijohnson-NOAA I will work on this today and tomorrow. Thank you for the reminder. I will be able to complete by 7/30.

**Comment by brianlangseth-NOAA** [comment]
@kellijohnson-NOAA I just updated the column widths and cleaned up some of the naming. I was able to test that these changes worked on a test document, so hope they work on the actual document. 

We (I) have not added Ntows for rec fleets, HKL, and Lam research; nor broken out comm_discards by state (#89). I can easily do so for Lam research, and awaiting a response from Harms on how to designate trips for the HKL survey. I plan to add those two into the tables but these should not hold back submission.

**Comment by brianlangseth-NOAA** [comment]
@kellijohnson-NOAA I added the commercial discard length samples to the sample size table that is broken down by state for the north model, in dbee8d4. Thank you to @andi-stephens-NOAA for getting me the state specific information. 

I also adjusted the formatting of the table based on @chantetwetzel-NOAA dover tables (and now it is much more streamlined). 

Ive decided not to add Ntows for non-expanded comps, so HKL, Lam research, and rec fleet do not have the tow/trip/drop level information. This was to maintain consistency among those three data sources. Also the HKL "trip" level definition is not all that well defined. I will update #89 with language describing how to do so for Lam research and HKL should our future selves be required, or want to do so.
